ext-src/models/IqComponentModel.ts

```diff
@@ -181,6 +181,8 @@ export class IqComponentModel implements ComponentModel {
               resolve();
             }).then(() => {
               if (!this.reportUrl.startsWith(this.url)) {
+                // remove trailing slash if it exists
+                this.url = this.url.replace(/\/$/, "")
                 this.reportUrl = `${this.url}/${this.reportUrl}`;
```

Comment:
You could just do this up in the constructor where we set it, that way it goes clean to IQ as well. If not you can also do:

`this.reportUrl = `${this.url.replace(/\/$/, "")}/${this.reportUrl}`;`